src/WebJobs.Script/Description/DotNet/DotNetFunctionDescriptorProvider.cs

```diff
@@ -154,6 +155,12 @@ protected override IFunctionInvoker CreateFunctionInvoker(string scriptFilePath,
                     descriptors.Add(new ParameterDescriptor(ScriptConstants.SystemExecutionContextParameterName, typeof(ExecutionContext)));
                 }
 
+                if (!descriptors.Any(p => p.Type == typeof(CancellationToken)))
+                {
+                    // Add the CancellationToken. We need this to ensure we can cancel even if the Function hasn't requested it.
```

Comment:
This injection of a cancellation token isn't needed anymore, right? as per our discussions